drop-cap/library/src/main/java/com/novoda/dropcap/DropCapView.java

```diff
@@ -126,16 +129,18 @@ public void setCopyFontType(String fontPath) {
 
         hasPaintChanged = true;
         requestLayout();
+        invalidate();
     }
 
-    public void setDropCapTextSize(@DimenRes float textSize) {
+    public void setDropCapTextSize(float textSize) {
```

Comment:
`float` should imply pixels. Providing `dp` would not make much sense because it will force you to evaluate the value in pixels later. Another option is to leverage resources instead and have `@DimenRes int textSizeResId` as parameter. But in that case maybe you could consider just an overload for `setDropCapTextSize`.